.circleci/config.yml

```diff
@@ -53,6 +54,11 @@ jobs:
                     python --version
                     npm run test
 
+          - run:
+                name: Percy TearDown
```

Comment:
there are numerous requests on circleci community about conditional `Jobs` upon previous failure. right now, the behavior is when the previous job fails, the percy finalize job cannot be executed.